.github/PULL_REQUEST_TEAMPLATE.md

```
@@ -0,0 +1,17 @@
+<!-- 이 PR과 연결된 이슈 번호를 명시해주세요 (예: Close #123) -->
+- Close #
+
+## 📙 작업 설명
+<!-- 주요 수정 사항이나 개발 내용을 요약해주세요 -->
+-
+
+## 📸 스크린샷 또는 시연 영상 (선택)
+<!-- UI 변경사항이 있다면 이미지나 GIF를 첨부해주세요 -->
+|기능|미리보기|기능|미리보기|
+|:--:|:--:|:--:|:--:|
+| 기능 설명 |<img src="링크" width="300" />| 기능 설명 |<img src="링크" width="300" />|
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)